### PR TITLE
Add new workflow to create a PR to update autofill automatically

### PR DESCRIPTION
**Pull request by marcosholgado**
Task/Issue URL: https://app.asana.com/0/488551667048375/1203205916393903/f 

### Description
This PR adds a new job into the update internal libraries workflow to also create a PR to update autofill

### Steps to test this PR
Check the PR created by the workflow and make sure the result is as expected and assigned to @shakyShane  -> https://github.com/duckduckgo/Android/pull/2433

### CODE REVIEW

**Comment by marcosholgado** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #2428** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/2428" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/2428?utm_source=stack-comment).

**Comment by marcosholgado** [comment]
@CDRussell, @shakyShane  assigning to you both to review this PR to automate autofill PR updates.

File: .github/workflows/update_internal_libraries.yml [hunk 1]
```diff
@@ -59,3 +60,55 @@ jobs:
...
+
+      - name: Install copy-files-from-to
+        run: npm install -g copy-files-from-to
+
+      - name: Check for updates
+        id: update-check
+        continue-on-error: true
+        run: ncu @duckduckgo/autofill --errorLevel 2
+
+      - name: Update reference tests
```
Comment (CDRussell) [inline .github/workflows/update_internal_libraries.yml:85]: Not sure what these two steps each specifically do ("Check for updates" and "Update reference tests") but I assume

- both are correct
- the latter needs renamed

@marcosholgado can you rename/clarify?

**Review by CDRussell — APPROVED** [review]
Workflow untested but looks good as far as I can see 👍

File: .github/workflows/update_internal_libraries.yml (hunk 1 above)
Comment (marcosholgado) [reply .github/workflows/update_internal_libraries.yml]: Check for updates as its name explains checks if there are any new versions for autofill. The latter needs renaming and I've just done it :)

Comment (marcosholgado) [reply .github/workflows/update_internal_libraries.yml]: > Check for updates as its name explains checks if there are any new versions for autofill so if not the workflow finished there. The latter needs renaming and I've just done it :)

**Comment by marcosholgado** [comment]
Closing in favour of what @GioSensation is working on :)

**Pull request closed without merging**